rakelib/artifacts.rake

```diff
@@ -101,18 +101,6 @@ namespace "artifact" do
     @exclude_paths << 'vendor/**/gems/**/Gemfile.lock'
     @exclude_paths << 'vendor/**/gems/**/Gemfile'
 
-    @exclude_paths << 'vendor/jruby/lib/ruby/gems/shared/gems/rake-*'
```

Comment:
These are no longer required as they are handled by the new `plugin:clean-duplicate-gems` task.